Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MKMapViewTests.swift
+//  SwifterSwift
+//
+//  Created by Hannes Staffler on 24.01.19.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(MapKit)
+import MapKit
+
+import struct CoreLocation.CLLocationCoordinate2D
+
+final class MKMapViewTests: XCTestCase {
+
+    func testRegisterAndDequeue() {
```

Comment:
We generally want 1 test per function, although here it kind of makes sense to combine them. Nevertheless, can you please split them into 2 functions, `testDequeue` and `testRegisterAndDequeue`, where the first one tests just the `dequeueReusableAnnotationView(withClass:)` function so that it doesn't need the `@available` attribute.